### PR TITLE
Update dependencies.props

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

**Comment by sebastienros** [comment]
😢

File: build/dependencies.props [hunk 1]
```diff
@@ -2,120 +2,121 @@
   <PropertyGroup Label="Package Versions: .NET Core dependencies">
 
     <!-- Packages from dotnet/core-setup -->
-    <MicrosoftExtensionsDependencyModelPackageVersion>3.0.0-preview-27122-01</MicrosoftExtensionsDependencyModelPackageVersion>
-    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview-27122-01</MicrosoftNETCoreAppPackageVersion>
-    <MicrosoftNETCoreDotNetAppHostPackageVersion>3.0.0-preview-27122-01</MicrosoftNETCoreDotNetAppHostPackageVersion>
-    <MicrosoftDotNetPlatformAbstractionsPackageVersion>3.0.0-preview-27122-01</MicrosoftDotNetPlatformAbstractionsPackageVersion>
+    <MicrosoftExtensionsDependencyModelPackageVersion>3.0.0-preview-27204-02</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview-27204-02</MicrosoftNETCoreAppPackageVersion>
```
Comment (natemcmaster) [inline build/dependencies.props]: Try updating this group to `3.0.0-preview-27206-02`

**Review by natemcmaster — APPROVED** [review]
It looks like the build is passing.

**Comment by sebastienros** [comment]
It's failing on Libuv tests, I asked @halter73 to take a look

**Pull request closed without merging**